cachi2/core/package_managers/bundler/parser.py

```diff
@@ -38,19 +40,32 @@ class _GemMetadata(pydantic.BaseModel):
     name: str
     version: str
 
+    def download_to(self, fs_location: Path) -> None:
+        return None
+
```

Comment:
I would add this to a separate commit or even drop it (since it currently does nothing) until we start adding all functions together.

Reply:
This is the default implementation, I _think_ it will remain unchanged for PathDependency. I can split it out though if this helps.

Reply:
I prefer keeping it simple unless we really really need extra complexity (e.g. when it makes things simpler someplace else). Moreover, this is not an interface spec, this is actual default fallback that will be executed. I would have returned `NotImplemented` or raised if I intended it to be an interface. This method exists solely for side-effects, and by default there is no side effect.